tests/modules/programs/thunderbird/thunderbird.nix

```diff
@@ -35,12 +37,18 @@
     };
   };
 
+  # Confirm that both Firefox and Thunderbird can be configured at the same time.
+  programs.firefox = { enable = true; };
```

Comment:
<3

Reply:
:) On second thought, maybe we should have a separate test case for both installed. Otherwise we may miss a bug in the future where Thunderbird would rely on something that Firefox module implements, and would fail otherwise.